api/v1alpha1/ramenconfig_types.go

```diff
@@ -105,6 +105,18 @@ type RamenConfig struct {
 	// MaxConcurrentReconciles is the maximum number of concurrent Reconciles which can be run.
 	// Defaults to 1.
 	MaxConcurrentReconciles int
+
+	// dr-cluster operator configuration
+	DrClusterOperator struct {
```

Comment:
We want to override `Subscription` configuration with values, and hence should consider the following configuration values here:
```
    channel: [default: alpha]
    name: [default: ramen-dr-cluster-operator] (needs configuration so that product streams can configure it appropriately)
    source: [default: ramen-catalog]
    sourceNamespace: [default: ramen-system]
    startingCSV: [default: ramen-dr-cluster-operator.v0.0.1]
``` 

The configurations can *ideally* be derived using the subscription at the hub in the ramen namespace and the namespace can default to the same namespace as the current namespace at the hub. But that can possibly come later and for now we go with the defaults.

Reply:
Ok.  I like the idea deriving them from hub configuration (which is what you said previously in a standup meeting!) and it replacing the ramen dr-cluster operator config map inputs altogether.  For the case where hub operator is installed from ramen operator container image, not bundle, say built from source, and user wants to specify non-default values, I suppose a subscription object would be required, and it'd probably need to be valid;  Hence hub installation from bundle would be required to specify non-default dr-cluster config values.

Reply:
I wonder if there is a use case where hub and dr-cluster are to be configured differently...such that deriving from hub configuration is not what is desired for the dr-cluster...

Reply:
Agree, I think the order should be user-defined fields in the configMap first, Subscription next (if available) and defaults last as the backup.

Reply:
Also, there is only one set of configuration values for all dr-clusters managed by a hub operator...I wonder if some dr-clusters should have different values than others...if all clusters in a dr-policy are the same....or if each dr-cluster needs its own configuration?  Also, if configuration is moved out of config map to say dr-policy then it likely becomes part of api...

Reply:
`Subscription`'s `CatalogSource` `name` and `namespace` were added in https://github.com/hatfieldbrian/ramen/commit/b53c30c0f239bc1a734f42b74a865c21bfe5dbe9.  The `ClusterServiceVersion` `name` input is there too.  An issue is needed to track adding the channel name and ramen dr-cluster operator name inputs.

Reply:
Issue #364 tracks deriving dr-cluster subscription values from hub subscription.